### PR TITLE
Appease eslint: semicolons and such.

### PR DESCRIPTION
**Pull request by marisaleung**
This change is [Reviewable](https://reviewable.io/reviews/google/blockly/1124)

### CODE REVIEW

File: core/field_variable.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ Blockly.FieldVariable.dropdownCreate = function() {
   }
   options.push([Blockly.Msg.RENAME_VARIABLE, Blockly.RENAME_VARIABLE_ID]);
   options.push([Blockly.Msg.DELETE_VARIABLE.replace('%1', name),
-               Blockly.DELETE_VARIABLE_ID]);
+    Blockly.DELETE_VARIABLE_ID]);
```
Comment (AnmAtAnm) [inline core/field_variable.js]: This should be at least a 4-space indent, and large indentation for syntactic sibling (as it was before) is very common, if not preferred.

File: core/workspace.js [hunk 2]
```diff
@@ -215,8 +215,8 @@ Blockly.Workspace.prototype.updateVariableStore = function(clear) {
     // Get variable model with the used variable name.
     var tempVar = this.getVariable(name);
     if (tempVar) {
-       varList.push({'name': tempVar.name, 'type': tempVar.type,
-                     'id': tempVar.getId()});
+      varList.push({'name': tempVar.name, 'type': tempVar.type,
+        'id': tempVar.getId()});
```
Comment (AnmAtAnm) [inline core/workspace.js]: Same.

File: core/field_variable.js (hunk 1 above)
Comment (rachel-fenichel) [inline core/field_variable.js]: 4: https://google.github.io/styleguide/jsguide.html#formatting-indent

Comment (marisaleung) [inline core/field_variable.js]: Done

File: core/workspace.js (hunk 2 above)
Comment (marisaleung) [inline core/workspace.js]: Done

**Comment by marisaleung** [comment]
screen shot 2017-05-26 at 2 29 32 pm

**Pull request merged**